roles/pulp3-devel/tasks/main.yml

```diff
@@ -0,0 +1,96 @@
+---
+- block:
+
+  - name: Install several useful packages
+    package:
+      name:
+        - bash-completion
+        - dnf-utils
+        - dstat
+        - fd-find
+        - fedora-easy-karma
```

Comment:
Can we remove this? I don't many of these tools, can we reduce this to the minimum of tools that are used by the aliases for now, and any tools that others want can be an additive change later?